src/app/public/modules/phone-field/types/country.ts

```diff
@@ -1,9 +1,32 @@
 export interface SkyPhoneFieldCountry {
+  /**
+   * Specifies the name of the country.
+   * @required
```

Comment:
I added the `@required` property, but it didn't add the required label to the descriptions or put the required properties before the optional ones. What do we need to do to for that. Do we need to file a docs-tools issue? Or do I need to do something else in this PR? @Blackbaud-SteveBrush

Reply:
Talked this over with @Blackbaud-AlexKingman. We'll bring this up with the team to determine if we need a docs-tools issue, but it's fine for now.